bigtable/benchmarks/embedded_server_test.cc

```diff
@@ -13,13 +13,10 @@
 // limitations under the License.
 
 #include "bigtable/benchmarks/embedded_server.h"
-
-#include <thread>
-
 #include <gmock/gmock.h>
-
-#include "bigtable/admin/table_admin.h"
+#include <thread>
```

Comment:
Header in wrong order

Reply:
Same.

Reply:
Given that we specify header order in the style guide can we use either of the following constructs to help clang-format do the right thing?

1. Expect user to divide headers into different blocks seperated by line and letting clang-format handle the sorting. This can be achieved by setting the "IncludeBlocks" directive to IBS_Preserve.

2. Using "IncludeCategories" directive to define regexes so that clang-format comes up with both the block seperation of headers and the sorting of those blocks. 

My vote would be for 1. 

Reference for clang-format directives is here -> https://clang.llvm.org/docs/ClangFormatStyleOptions.html

Reply:
Yes we can configure clang-format to do that.  We haven't because when we do the CI builds will break (clang-format is already enforce some order, not exactly the one we want).  First we would need to merge #331.

On a side note, `IncludeBlocks` was introduced in Clang 6, we are using Clang 4 to do the formatting.  If we go that way we should do it in a separate PR introducing a build for that version of the compiler + tools.